future_proofing.md

```diff
@@ -1,21 +1,33 @@
 ## Future-Proofing
 
-- Versioning of the standard
+### Versioning
 
-- Extensibility
+This version of the standard is known as "DIDComm v2" &mdash; acknowledging the fact that a [v1 generation of DIDComm specs](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0005-didcomm/README.md) was incubated under the Hyperledger Aries project umbrella. The v1 specs are close conceptual cousins, but use a slightly different encryption envelope, and base their plaintext format on arbitrary JSON instead of JWMs.
 
-- Adapting for post-quantum crypto
+Future evolutions of the spec will follow [semver](https://semver.org) conventions. Minor updates that add features without breaking compatibility will carry a minor version number update: 2.1, 2.2, and so forth. Breaking changes will prompt a major version change.
 
 ### Extensions
 
-The general mechanism for DIDComm extensibility is the development of
-DIDComm protocols. In the case where extensibility requires a modification
-to the base DIDComm spec itself, a DIDComm Extension is to be used.
+The general mechanism for DIDComm extensibility is the development of DIDComm protocols. In the case where extensibility requires a modification to the base DIDComm spec itself, a DIDComm extension is to be used. An extension adds a self-contained set of conventions or features. Support for DIDComm extensions is optional.
 
-Each DIDComm Extension is described in a specification. Software that
-implements a DIDComm Extension in addition to the DIDComm spec will
-indicate so via link to the extension spec.
+Each DIDComm extension is described in a specification of its own. Software that implements a DIDComm Extension in addition to the DIDComm spec will indicate so via link to the extension spec.
 
-### Encoding
+### Future Work
 
-DIDComm Messages are JSON encoded (based on the JOSE family of specs) at the encryption, signature, and content level. Future encodings MUST specify a deterministic and reliable method for indicating the alternative encoding used.
\ No newline at end of file
+#### Additional Encodings
+
+DIDComm Messages are JSON encoded (based on the JOSE family of specs) at the encryption, signature, and content level. Future encodings might introduce binary serializations. Each innovation like this MUST specify a deterministic and reliable method for indicating the alternative encoding used.
+
+### Beyond Messaging
```

Comment:
This is a good addition in my opinion and I think it would be wise to keep it in here no matter the outcome of the discussion on Post-Quantum Crypto.